cypress/support/Helpers/apiHelper.ts

```diff
@@ -1,28 +1,54 @@
+import CandidatePayloadInit from "../Initializers/CandidatePayloadInit";
 import UserPayLoadInit from "../Initializers/UserPayloadInit"
-export enum StatusCode{
- Success = 200,
+export enum StatusCode {
+    Success = 200,
 }
-export  const Urls ={
-    employees: `${Cypress.env("apiLogin")}/employees`
+export const Urls = {
+    employees: '/web/index.php/api/v2/employees',
+    viewCandidates: '/web/index.php/api/v2/recruitment/candidates?limit=50&offset=0&model=list&sortField=candidate.dateOfApplication&sortOrder=DESC',
+    addCandidate: '/web/index.php/api/v2/recruitment/candidates',
+    shortlistCandidate: '/web/index.php/api/v2/recruitment/candidates/',
+    viewCandidateProfile: '/web/index.php/recruitment/addCandidate/'
 }
-export class apiHelper{
-    static addNewEmployee() {
-    return cy.fixture("employeeInfo").then(employeeData => {
-        cy.request({
+export class apiHelper {
```

Comment:
PascalCase, and it's better to have a useful name, like it can be DataUtils since you are dealing with data